modules/server_config/outputs.tf

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "forseti-server-config" {
+  description = "The rendered Forseti server configuration file"
+  value       = data.template_file.forseti_server_config.rendered
+}
+
+output "forseti-server-config-md5" {
```

Comment:
What are these outputs used for?

Reply:
They are used to prevent the root module from trying to pull down the config for GKE before it gets created in the GCS bucket.

https://github.com/forseti-security/terraform-google-forseti/blob/c73c874f833afb2759db08e09271707332b5c50f/modules/on_gke/main.tf#L104-L118

Reply:
Got it- thanks!